tests/subsys/bluetooth/fast_pair/storage/account_key_storage/src/main.c

```diff
@@ -34,53 +38,52 @@ static void before_fn(void *f)
 
 	int err;
 
-	cu_account_keys_validate_unloaded();
+	cu_account_keys_validate_uninitialized();
 
 	/* Load empty settings before the test to initialize fp_storage. */
 	err = settings_load();
-
 	zassert_ok(err, "Settings load failed");
+
+	err = fp_storage_manager_init();
+	zassert_ok(err, "Failed to initialize module");
 }
 
 static void after_fn(void *f)
 {
 	ARG_UNUSED(f);
 
 	fp_storage_ak_ram_clear();
+	fp_storage_manager_ram_clear();
```

Comment:
I'd rather not overcomplicate tests. We already don't have time for writing tests and end up with no tests at all. Tests should be simple.

Reply:
To be honest, that would significantly simplify tests and let us to avoid exposing most of private APIs to clear RAM. @kapi-no what do you think? Maybe we can make it as a follow-up task?